internal/subshell/subshell.go

```diff
@@ -68,6 +66,9 @@ type SubShell interface {
 	// RcFile return the path of the RC file
 	RcFile() (string, error)
 
+	// EnsureRcFile ensures that the RC file exists
+	EnsureRcFile() error
```

Comment:
Can we name this `EnsureRcFileExists`? This has me going.. "ensure RC file .. and then?" :p

![image](https://user-images.githubusercontent.com/278649/198686582-e9a2b485-4981-42f6-a57c-42334fec8f83.png)